docs/blog_post.md

```diff
@@ -0,0 +1,56 @@
+
+![](human_example.JPG)
+<!--TK image caption formatting-->
+The Urban Wildlife Institute's cameras often capture images of people jogging or walking dogs, or (as in this case) images of the researchers themselves stopping by to service them.
+
+To address this problem, Uptake's philanthropic arm Uptake.org developed the first version of a program called Autofocus that can classify an image as containing a human or not. That program proved to be sufficiently accurate to remove approximately 50% of human-containing images with a negligible rate of false positives. **On one batch of images from Spring 2016, the Urban Wildlife Institude was able to use that model to filter out roughly 3,700 images.** "It was like they gifted us 10 interns who could work around the clock," said Urban Wildlife Institute researcher Mason Fidino. "Autofocus enables us to get on top of our backlog of images and deliver valuable insights on local wildlife now."
+
+**Data scientists at ShopRunner contributed to a second version of Autofocus that not only appears to increase human label accuracy, but also provides labels for eighteen additional kinds of animals.** The accuracy of that second version is currently under review, but preliminary results suggest that it is sufficient to automate classification of a large percentage of incoming images at least for common animal types such as deer and raccoons, further improving the speed of the Urban Wildlife Institute's research.
+
+## Methods
+
+![](https://brooksbrothers.scene7.com/is/image/BrooksBrothers/SX00251_HEATHER-GREY?$bbproductimages$)
+<!--TK get a fashion image that we definitely have permission to use and adjust caption accordingly-->
+<!--TK image caption formatting-->
+ShopRunner data scientists repurposed for Autofocus work that they did to develop a system that can classify this image as showing a gray, melange, knee-length, sleeveless dress for ShopRunner's iOS shopping app District.
+
+The Autofocus image classification model uses techniques from *deep learning*, an area of artificial intelligence that allows computers to learn how to perform tasks such as labeling images from examples rather than through explicit rules. Deep learning models for image classification have been known to provide greater accuracy than trained humans on public benchmark datasets.
+
+ShopRunner uses deep learning image classification along with text classification models internally to place products from across its large retailer network into one set of categories so that users of its iOS app District can shop across that network in one convenient interface. Applying the same techniques to Autofocus allows its data scientists to hone their skills further in a way that has the potential to foster healthy ecosystems.
```

Comment:
As mentioned earlier, these categories are used in other places, for example on our website and in emails. It's good to call out District, but it would also be nice to highlight how crucial this work is to the entire ShopRunner ecosystem.